addon/components/frost-list.js

```diff
@@ -167,19 +183,24 @@ export default Component.extend({
       this.onExpansionChange(this.get('items'))
     },
 
+    _onPaginationChange () {
```

Comment:
Originally I was going to write a action that fired on pagination change that would reset the anchors. I then realized the more I stared at the flow of the code, this cause extra complexity and I forgot to remove it before I moved onto a different route. Good catch though.